### DISCUSSION TITLE
Copilot locks dockerfile preventing starting of docker containers in Rider

### DISCUSSION THREAD

**Discussion by martin-gft** [community/community#112092] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Since I've upgraded the Github Copilot Rider Plugin from 1.4.* to 1.5.0.5148 starting a docker container in Rider 2023.3 doesn't work anymore. Copilot locks the docker file that needs to be adjusted by Rider.

`Building image…
Preparing build context archive…
[>                                                  ]0/570 files
Failed to deploy 'XX Dockerfile: XX/Dockerfile': java.io.IOException: The process cannot access the file because another process has locked a portion of the file
`

Disabling Copilot solves this problem. The file is not locked by any other process. Starting everything using Visual Studio works fine.

**Comment by SpartaDeveloper93** [comment]
I am having the same problem
